src/lammpsio/topology.py

```diff
@@ -150,6 +153,21 @@ def num_types(self, value):
         else:
             self._num_types = None
 
+    @property
+    def label(self):
```

Comment:
I would prefer to call these properties `type_label` or `typeid_label` to be consistent with the LAMMPS nomenclature. LAMMPS calls its numerical values `type`, but we use `typeid` to make clear that it is indeed just a number.

Also, is the snapshot missing a similar property?